starter/script.js

```diff
@@ -1 +1,71 @@
+        if (whatNow == "stay"){
+          return poolHallDrinks();
+        } else return sadWalk();
+    } else if (poolHall == "go to the jukebox") {
+      alert ("You make your way confidently to the TouchTunes jukebox by one of the pool tables, beneath the FC Barcelona poster.");
+      return jukeboxHero();
+    } else if (poolHall == "order a beer") {
+      return howDrunkToGet();
+    } else alert(". . . Seriously? Come on!")
+      return poolHallDrinks();
+};
+
+var sadWalk = function() {
+  alert("You get outta Dodge in a hurry, but in your haste to leave, you trip on the threshold of the bar's entrace. All the patrons laugh at your clumsiness, of course. You hang your head and amble home, mumbling nonsense to yourself about your poor life choices.");
+};
+
+var jukeboxHero = function() {
+  var songChoice = prompt("Clearly, at this point, you have one of two choices: Gordon Lightfoot or some garbage reggaeton that the people in the bar seem to like. Which will it be?" , "Lightfoot, reggaeton, or wild card");
+  if (songChoice == "Lightfoot") {
+    alert("You cue up The Wreck of the Edmund Fitzgerald by Gordon Lightfoot and indulge in a spirited singalong. The patrons are not impressed and return to their beers. You decide to head to the bar for a drink, but the owner decides to refuse you service. You head home, dejected. Another successful night out, amirite!");
+  } else if (songChoice == "reggaeton") {
+    alert("You pick an album on the jukebox with a Puerto Rican flag and scantily clad woman on the cover. The patrons seems to dig it, so they ask you over to the bar to engage in a singalong. You make friends with everyone, and drink a whole bunch of Modelos. The rest of the evening is a blur, which means . . . great success!");
+  } else alert("The machine eats your money but doesn't let you play any music, so you start cursing loudly and mumbling something about wishing you were back home in Wisconsin. Unimpressed by your tantrum, the patrons return to their games of pool and beers. You ask the owner if he's a Packers fan, and the guy just stares. You spend the next 40 minutes delivering an encomium to Brett Favre, at which point you black out. If you can't remember it, it must have been fun!")
+};
+
+var howDrunkToGet = function() {
+  var howManyDrinks = parseInt(prompt("You belly up to the bar and order some beers. How many will you have?"));
+  while (howManyDrinks <= 6) {
+    alert("Never one to be bashful about lager, you guzzle your " + howManyDrinks + " beer(s) and order more." )
+    howManyDrinks += 2;
+  }
+  alert("Well, you've already been drinking for eight hours, so it's about time to call it a night. The owner of the pool hall appears to appreciate your patronage—he even invites you back to watch the Notre Dame game on Saturday. You head home, pleased with yourself. Another successful night on the town!");
+};
+
+boozeMore();
```

Comment:
How would you go about dealing with user inputs that **almost** match your `if` and `else if` statements (ex. what if the user submits a correct response but in all caps)?